R/zchunk_L231.proc_sector.R

```diff
@@ -0,0 +1,460 @@
+             "L1322.in_EJ_R_indfeed_F_Yh",
+             "L1322.in_EJ_R_indenergy_F_Yh"))
+  } else if(command == driver.DECLARE_OUTPUTS) {
+    return(c("L231.UnlimitRsrc",
+             "L231.UnlimitRsrcPrice",
+             "L231.FinalDemand_urb",
+             "L231.Supplysector_urb_ind",
+             "L231.SubsectorLogit_urb_ind",
+             "L231.SubsectorShrwt_urb_ind",
+             "L231.SubsectorShrwtFllt_urb_ind",
+             "L231.SubsectorInterp_urb_ind",
+             "L231.SubsectorInterpTo_urb_ind",
+             "L231.StubTech_urb_ind",
+             "L231.GlobalTechShrwt_urb_ind",
+             "L231.GlobalTechEff_urb_ind",
+             "L231.GlobalTechCoef_urb_ind",
+             "L231.GlobalTechCost_urb_ind",
+             "L231.RegionalTechCalValue_urb_ind",
+             "L231.IndCoef"))
+  } else if(command == driver.MAKE) {
+
+    all_data <- list(...)[[1]]
+
+    # Load required inputs
+    GCAM_region_names <- get_data(all_data, "common/GCAM_region_names")
+    A_regions <- get_data(all_data, "emissions/A_regions")
+    A31.rsrc_info <- get_data(all_data, "emissions/A31.rsrc_info") %>%
+      gather(year, value, matches(YEAR_PATTERN)) %>%
+      mutate(year = as.integer(year))
+    A31.sector <- get_data(all_data, "emissions/A31.sector")
+    A31.subsector_logit <- get_data(all_data, "emissions/A31.subsector_logit")
+    A31.subsector_shrwt <- get_data(all_data, "emissions/A31.subsector_shrwt")
+    A31.subsector_interp <- get_data(all_data, "emissions/A31.subsector_interp")
+    A31.globaltech_shrwt <- get_data(all_data, "emissions/A31.globaltech_shrwt") %>%
+      gather(year, share.weight, matches(YEAR_PATTERN)) %>%
+      mutate(year = as.integer(year))
+    A31.globaltech_eff <- get_data(all_data, "emissions/A31.globaltech_eff") %>%
+      gather(year, efficiency, matches(YEAR_PATTERN)) %>%
+      mutate(year = as.integer(year))
+    A31.globaltech_cost <- get_data(all_data, "emissions/A31.globaltech_cost") %>%
+      gather(year, input.cost, matches(YEAR_PATTERN)) %>%
+      mutate(year = as.integer(year))
+    A31.globaltech_coef <- get_data(all_data, "emissions/A31.globaltech_coef") %>%
+      gather(year, coefficient, matches(YEAR_PATTERN)) %>%
+      mutate(year = as.integer(year))
+    A32.globaltech_eff <- get_data(all_data, "energy/A32.globaltech_eff") %>%
+      gather(year, value, matches(YEAR_PATTERN)) %>%
+      mutate(year = as.integer(year))
+    L1322.in_EJ_R_indfeed_F_Yh <- get_data(all_data, "L1322.in_EJ_R_indfeed_F_Yh")
+    L1322.in_EJ_R_indenergy_F_Yh <- get_data(all_data, "L1322.in_EJ_R_indenergy_F_Yh")
+
+    # ===================================================
+    # L231.FinalDemand_urb: Final demand information for urban processes sector
+    L231.FinalDemand_urb <- tibble(
+      region = A_regions$region,
+      energy.final.demand = "urban processes",
+      perCapitaBased = 1,
+      income.elasticity = 0,
+      base.service = 0.004,
+      aeei = 0
```

Comment:
What is `aeei`? Where units of these values?

Reply:
I'm not sure, but Google tells me "Autonomous Energy Efficiency Improvement (AEEI) parameter"

A lot of the values in this chunk are pretty meaningless to me, as there are mostly just a whole lot of 0's and 1's.